ops/testing.py

```diff
@@ -394,9 +394,30 @@ def _create_meta(self, charm_metadata: Optional[YAMLStringOrFile],
         """Create a CharmMeta object.
 
         Handle the cases where a user doesn't supply explicit metadata snippets.
+        Will try to load metadata/config from charm_dir/charmcraft.yaml first, then
+        charm_dir/metadata.yaml and/or charm_dir/actions.yaml if charmcraft.yaml
+        not include metadata and/or actions.
```

Comment:
It looks like this method doesn't deal with config. Also a few minor tweaks for English and clarity. Is this correct?

```suggestion
        This will try to load metadata from ``<charm_dir>/charmcraft.yaml`` first, then
        ``<charm_dir>/metadata.yaml`` if charmcraft.yaml does not include metadata,
        and ``<charm_dir>/actions.yaml`` if charmcraft.yaml does not include actions.
```